### ISSUE TITLE
Add pipe system

### ISSUE THREAD

**Issue by clement-or** [space-wizards/space-station-14#17753]
## Description

Following PR #14348, for now sinks are magical beings who generate and destroy water as they please. It would be nice having a pipe/sewers system that brings water to places in the station and disposes of waste. This would bring new gameplay for janitors and would be an excuse for more mishaps.

Liquid pipes should work somewhat similar to gas pipes I suppose. They'd accumulate reagents over time depending on what is put inside them. Some kind of goggles or tool could allow a janitor to know if a pipe is clogged or not, and they would need to fasten the pipes connected to the clogged one in order to replace or clean it.

Pipes could get damaged, because of mice, random events, sabotage. Someone could replace the water supply with beer (or even poison, make sure it's not OP).

**Comment by PJB3005** [comment]
I feel like the problem is going to be that there simply _aren't enough things_ that would need plumbing.

**Comment by forgotmyotheraccount** [comment]
> I feel like the problem is going to be that there simply _aren't enough things_ that would need plumbing.
This

Sinks being magic water generators is mid but until there's a bunch of stuff besides sinks that would make use of plumbing, its not worth the effort. I guess you could flush stuff down toilets?

**Comment by clement-or** [comment]
Opportunity to add stuff, like decontamination showers that Bay used to have, fire hydrants, and so on

**Comment by moonheart08** [comment]
I dont think there's going to be nearly enough stuff to justify pipes.

**Comment by PJB3005** [comment]
I do have to say I wouldn't *mind* having another set of pipes in maintenance. Just saying. Muh immersion.

**Comment by Beridot** [comment]
I want plumbing so that when infinichem gets gutted we can have giant vats of chemicals.

**Comment by PJB3005** [comment]
(I should think about comments more before posting.)

That said though, the real risk with adding a niche-case system like this is that it won't see enough use, and then the jobs that DO rely on it just get FUCKED when it gets damaged/sabotaged mid-round and engineers don't care.

Imagine if you can't do botany anymore because the plumbing is out and you don't have water. Ugh. I know Goon also has this with the DWAINE system and science.

**Comment by AJCM-git** [comment]
Disposals *could* be pontentially purposed as a pipe system too with some thought, could make the rides there more fun since you could drown and showering in literal trash water seems appropriate for the game

**Comment by clement-or** [comment]
Most jobs relying on water (mainly chef and botany) should have (and already have) a water tank that should fulfill their need for an entire round.
And then they could order a new one. This would cut the monotony from the job a bit.

**Comment by Beridot** [comment]
also, water just requires oxygen and hydrogen, we can have salvage do gas collections and then turn it into water.

**Comment by TemporalOroboros** [comment]
That makes hot water. It could probably be routed through engineering for extra power in the process.

**Comment by PJB3005** [comment]
> Most jobs relying on water (mainly chef and botany) should have (and already have) a water tank that should fulfill their need for an entire round.

It seems a bit silly to have the jobs that'd rely MOST on this mechanic also need to have a permanent backup to not be dependent on it, due to inherent unreliability.

I mean, maybe that's a fine middle ground. Best of both worlds.

**Comment by clement-or** [comment]
> also, water just requires oxygen and hydrogen, we can have salvage do gas collections and then turn it into water.

Carbon-based lifeform privileges smh

**Comment by Emisse** [comment]
I think it should probably just be integrated with disposals lines so that it isn't mapping CBT and also gives more incentive for engis to fix since its a double pronged system. Also won't make subfloor so insanely cluttered.

**Comment by Peptide90** [comment]
Agreed mapping yet another pipe and cluttering subfloor would suck until we somehow get pipe layering.

I do also like the idea of plumbing. It doesn't need to be station wide. SS13 maps had a hot water tank in a maint cupboard that went a short way to its destination e.g. dorms showers. We don't need the whole station plumbed just areas that need it. Showers, washing machines and later game sprinklers could be uses (or a cool physics based fire hose and a hydrant as someone suggested could be fun ala Prison Architect, cooling and flooding areas).
